taking-turns.js

```diff
@@ -0,0 +1,241 @@
+//Variables shared between games
```

Comment:
You could wrap these  values in an initialize function which calls computerGuesses() inside itself to remove global variables.